gdb/PKGBUILD

```
@@ -2,7 +2,7 @@
 
 pkgname=gdb
 pkgver=10.2
-pkgrel=1
+pkgrel=2
 _gcc_ver=10.2.0
```

While bumping pkgrel could you also fix `_gcc_ver`?
```suggestion
_gcc_ver=10.3.0
```

---

MSYS2 gdb is still 10.2.0

---

Sorry, eyeballed wrong package.

---

I checked and mingw-w64-gdb does not seem to contain corresponding variable/file.  So, is that supposed to be mingw-w64-gcc version or msys2 gcc version?

---

It's correct right now. Github doesn't let me resolve this comment for some reason.